packages/swing-store-simple/simpleSwingStore.js

```diff
@@ -152,7 +167,7 @@ function makeStorageInMemory() {
  * }
  */
 function makeSwingStore(dirPath, forceReset = false) {
-  const { storage, state } = makeStorageInMemory();
+  const { storage, state, isDirty, markClean } = makeStorageInMemory();
```

Comment:
We should not be exposing `isDirty` and `markClean` on this API.  This API is shared with the LMDB store, for which these calls are not relevant, and anyway they are an internal implementation detail that shouldn't be opened for people to fiddle with.